### ISSUE TITLE
NVDA does not read the selected item in a WPF ComboBox upon expanding

### ISSUE THREAD

**Issue by hjihong** [nvaccess/nvda#12003]
### Steps to reproduce:
1. Create a WPF app that has an editable ComboBox consisting of a couple string items. 
2. Use keyboard shortcuts to open it and verify that each item is being read out loud. 
3. Select an item and ensure that the ComboBox is filled with the item's string. 
4. With keyboard focus on the box, ALT+down to expand the ComboBox. 
5. Once the ComboBox is expanded, the selected item should be highlighted. 

### Actual behavior:
NVDA does not read out the selected item. 

### Expected behavior:
NVDA should read out the title of the selected item. 

#### NVDA installed/portable/running from source:
NVDA installled.

#### NVDA version:
2020.3

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Visual Studio 16.8

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If addons are disabled, is your problem still occuring?
Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No.

**Comment by cary-rowen** [comment]
@hjihong Does this problem still exist?

**Comment by Adriani90** [comment]
We didn't get any update from the initial author upon request since many years. I am closing as abandoned. If you are still having this issue, please comment and we can reopen.
